featurebyte/query_graph/sql/feature_compute.py

```diff
@@ -68,6 +74,7 @@ def __init__(
             AggregationType.WINDOW: WindowAggregator(**aggregator_kwargs),
             AggregationType.ITEM: ItemAggregator(**aggregator_kwargs),
             AggregationType.AS_AT: AsAtAggregator(**aggregator_kwargs),
+            AggregationType.FORWARD: ForwardAggregator(**aggregator_kwargs),
```

Comment:
It looks like the forward aggregate specs are not being picked up. I think we need to add a condition here in `FeatureExecutionPlanner` to handle the new aggregation type:

https://github.com/featurebyte/featurebyte/blob/3f8ed5967b3362a0bd4771b71689cee8695bf414/featurebyte/query_graph/sql/feature_compute.py#L470-L488